packages/crypto/test/hmac.node.test.ts

```diff
@@ -0,0 +1,29 @@
+import { getBytes } from '@fuel-ts/utils';
+
+import { computeHmac } from '..';
+/**
+ * @group node
+ */
+describe('computeHmac node', () => {
+  it('should use the registered HMAC function', () => {
+    const key = '0x0102030405060708090a0b0c0d0e0f10';
+    const data = '0x11121314151617181920212223242526';
+    const expectedHmac = '0x1234567890abcdef';
```

Comment:
Were these test cases taken from `ethers`? If not how did you get the expected value?

Reply:
You have a variety of tools to derive the HMAC value such as [this](https://cryptotools.net/hmac) but this particular test is just evaluating that if we register an amended algorithm, that it will return the right value.